examples/kubernetes/README.md

```diff
@@ -0,0 +1,235 @@
+# Using the Cloud SQL proxy on Kubernetes
+
+The Cloud SQL proxy is the recommended way to connect to Cloud SQL, even when
+using private IP. This is because the proxy provides strong encryption and 
+authentication using IAM, which can help keep your database secure. 
+
+## Configure your application with Secrets
+
+In Kubernetes, [Secrets][ksa-secret] are a secure way to pass configuration 
+details to your application. You can create a Secret with details such as your
+database name, user, and password which can be injected into your application as
+env vars.
+
+1. Create a secret with information needed to access your database:
+    ```shell
+    kubectl create secret generic <YOUR-DB-SECRET> \
+        --from-literal=db_user=<YOUR-DATABASE-USER> \
+        --from-literal=db_password=<YOUR-DATABASE-PASSWORD> \
+        --from-literal=db_name=<YOUR-DATABASE-NAME>
+    ```
```

Comment:
From this block above, looks like a Secret is considered a bundle of key/val pairs together...that's a little muddy, but not sure about specifically how I'd clarify other than maybe just calling it out explicitly above like "A secret is a bundle of key/value pair configurations that you can inject into your application as env vars"